widget_button.go

```diff
@@ -40,10 +40,13 @@ func (w *ButtonWidget) Update(dev *streamdeck.Device) error {
 				w.fontsize,
 				image.Pt(-1, -1))
 		} else {
-			_ = drawImage(img, w.icon, height, image.Pt(-1, -1))
+			err = drawImage(img, w.icon, height, image.Pt(-1, -1))
 		}
 
-		err = dev.SetImage(w.key, img)
+		if err != nil {
+			return
+		}
```

Comment:
I'm not certain if that is what you wanted to achieve with these changes. But in case you define a button without an icon (not impossible) this results in the following not much saying error message (and `deckmaster` exits):
`2021/06/02 18:17:44 error: open : no such file or directory`

Reply:
Good catch! This is indeed just a side-effect of the missing error handling. I'll address this in a separate PR, which will improve the rendering in a few ways.